pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Landesfeind2019/electrolyte_conductivity_Landesfeind2019_EC_DMC_1_1.py

```diff
@@ -0,0 +1,31 @@
+import electrolyte_conductivity_Landesfeind2019_base as base
+import numpy as np
+
+
+def electrolyte_conductivity_Landesfeind2019_EC_DMC_1_1(c_e, T, T_inf, E_k_e, R_g):
```

Comment:
should we get rid of T_inf, E_k_E and R_g as inputs to these functions in general? When they are needed, they can easily be called from within the function as `pybamm.standard_parameters_lithium_ion.T_inf`, etc

Reply:
For R_g I would say yes. For T_inf could there be a situation where the reference temperature for different functions is different? We could also use **kwargs potentially although I don't know if that would work with the expression tree. But yeah if you see fewer problems I'm happy to make the changes

Reply:
I think we should definitely be open to the idea of different functional dependencies of these things, so being less prescriptive in terms of inputs is probably good

Reply:
if it's specific to a function then maybe it should be hard-coded into the function itself (same with E_k_e)

Reply:
Yeah maybe you are right, If there were a specific reference temperature it would probably be hardcoded into the function if there were other parameters dependent on that.